src/App.tsx

```diff
@@ -1,99 +1,13 @@
 /* eslint-disable max-len */
 import React from 'react';
-
+import listMovies from './api/movies.json';
```

Comment:
Consider renaming 'listMovies' to `movies` or `moviesFromServer` for better clarity.